docker-entrypoint.sh

```diff
@@ -53,7 +53,7 @@ curl -fail http://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem -o /
 # Allow for no passwords when running mysql as root...
 echo "[client]
 user=root
-password=$(cat ${ROOT_PASS_SECRET})
+password='$(cat ${ROOT_PASS_SECRET})'
```

Comment:
No it won't - it's not a bash file - it's a .my.cnf file and tested it on the actual container...